CHANGELOG.md

```diff
@@ -143,6 +153,11 @@ support clients for multiple versions of a GCP service from within the same
 library (e.g. `speech_v1::SpeechClient` and `speech_v2::SpeechClient`). See
 [#10170] for more details.
 
+- feat(datacatalog): generate lineage library ([#10977](https://github.com/googleapis/google-cloud-cpp/pull/10977))
```

Comment:
argh, just noticed this should probably have its own entry

Reply:
I thought not, given that it's only experimental.

Reply:
I forgot that it was experimental. I still don't think it belongs under "Common Libraries". I think we would either delete the entry, or list it under "Data Catalog".

But given that we don't suspect `datacatalog` has any users... I don't care, and would be happy to save some builds.

Reply:
Done (listed under "Data Catalog").